packages/foreman/foreman/foreman.spec

```diff
@@ -135,7 +135,7 @@ Requires: %{?scl_prefix}rubygem(get_process_mem)
 Requires: %{?scl_prefix}rubygem(rack-cors) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(rack-cors) < 1.1.0
 Requires: %{?scl_prefix}rubygem(jwt) >= 2.1.0
-Requires: %{?scl_prefix}rubygem(jwt) < 2.2.0
+Requires: %{?scl_prefix}rubygem(jwt) = 2.2.1
```

Comment:
Needs a bump in the BuildRequires as well

Reply:
Done. @ehelms it that correct?

Reply:
An exact match is usually incorrect. In general never pin this way. In general I use https://github.com/ekohl/theforeman-plumbing/blob/master/fm-packaging-update-deps to automatically do this.

Reply:
@ekohl would running this script require https://github.com/theforeman/foreman-packaging/pull/4182 to merged first?

Reply:
No, the core PR which has just been merged.